jena-text/src/main/java/org/apache/jena/query/text/assembler/TextIndexLuceneAssembler.java

```diff
@@ -144,6 +144,17 @@ public TextIndex open(Assembler a, Resource root, Mode mode) {
                 storeValues = svNode.asLiteral().getBoolean();
             }
 
+            // use query cache by default
+            boolean cacheQueries = true;
+            Statement cacheQueriesStatement = root.getProperty(pCacheQueries);
+            if (null != cacheQueriesStatement) {
+                RDFNode cqNode = cacheQueriesStatement.getObject();
+                if (! cqNode.isLiteral()) {
+                    throw new TextIndexException("text:cacheQueries property must be a string : " + cqNode);
```

Comment:
This is maybe a little picky, but maybe better: `"text:cacheQueries property must be a boolean`?

Reply:
Thanks. Best to be accurate.